File: RepoTransBench/target_projects/Java/Go/markzhai_DataBindingAdapter/public_tests/binding_viewholder_public_test.go
package public_tests

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

type dummyDataBindingPublic struct {
	Root any
}

func (d *dummyDataBindingPublic) GetRoot() any {
	return d.Root
}

func TestBindingViewHolderPublic_GetBindingPublic(t *testing.T) {
	binding := &dummyDataBindingPublic{Root: "view"}
	holder := NewBindingViewHolder(binding)
	assert.Equal(t, binding, holder.GetBinding())
}

File: RepoTransBench/target_projects/Java/C#/markzhai_DataBindingAdapter/public_tests/BindingViewHolderPublicTest.cs
using Xunit;
using MarkZhai.DataBindingAdapter;

namespace MarkZhai.DataBindingAdapter.Tests.Public
{
    public class BindingViewHolderPublicTest
    {
        public class DummyBinding { }

        private DummyBinding binding;
        private object view;

        public BindingViewHolderPublicTest()
        {
            binding = new DummyBinding();
            view = new object();
        }

        [Fact]
        public void TestGetBindingPublic()
        {
            var holder = new BindingViewHolder<DummyBinding>(binding);
            Assert.Same(binding, holder.GetBinding());
        }
    }
}